### DISCUSSION TITLE
Material from string

### DISCUSSION THREAD

**Discussion by zax71** [Minestom/Minestom#1608] [Help] [answered]
I have a String with the material syntax (E.G: OAK_BOAT), how would I convert that to a Material?

I use ~~Arch~~ Kotlin btw

**Comment by TheMode** [comment, marked as answer]
You can retrieve registry objects from namespace id using `#fromNamespaceId`, so `Material.fromNamespaceId`

**Reply by zax71** [reply]
For anyone coming across this (or my future self). You need `minecraft:oak_boat` instead of `OAK_BOAT`.
